src/database/schema.sql

```diff
@@ -0,0 +1,10 @@
+DROP DATABASE IF EXISTS todos;
+CREATE DATABASE todo;
```

Comment:
the scripts this project came with do the dropping and creating of the data base so remove these two lines and only create tables in this file.